rust/agama-lib/share/examples/storage/generate_pvs.json

```diff
@@ -0,0 +1,39 @@
+{
+  "storage": {
+    "drives": [
+      {
+        "alias": "first-disk"
+      },
+      {
+        "partitions": [
+          {
+            "alias": "pv1",
+            "id": "lvm",
+            "size": { "min": "10 GiB" }
+          }
+        ]
+      }
+    ],
+    "volumeGroups": [
+      {
+        "name": "system",
+        "physicalVolumes": [
+          "pv1",
```

Comment:
I suspect it would be easy, but I have to double check.

Reply:
I checked and it would be possible by modifying the yast2-storage-ng algorithm a bit. But we decided to leave it for later, as part of a separate task internally tracked at https://trello.com/c/XAaoYdCL/578-storage-profile-improve-support-for-generating-lvm-physical-volumes

Reply:
I have adapted the schema to allow either aliases or 'generate'.